spec/unit/lib/services/service_brokers/v2/http_client_spec.rb

```diff
@@ -161,6 +161,21 @@ module VCAP::Services::ServiceBrokers::V2
           end
         end
 
+        context 'when user_guid is specified as a parameter' do
```

Comment:
Should we also have a test that specifies that the parameter overrides the security-context? I cannot think of a scenario where the two are set and are different from each other tbh, would just be mostly for completeness.